src/vuln_analysis/utils/pre_triage.py

```
@@ -1166,7 +1169,8 @@ def _scan_go_binaries_with_strings(
     installations: list[PackageInstallation] = []
     for binary in go_binaries[:30]:
         try:
-            result = subprocess.run(
+            # Fixed strings argv against a discovered binary.
+            result = subprocess.run(  # nosec B603, B607
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)